starter-code/css/main.css

```diff
@@ -8,3 +8,94 @@ body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
 }
+
+.top-background{
+    background: linear-gradient(#6491c4, #1c5380);
+    height: 300px;
+}
+
+#instagram-brand{
+    position: relative;
+    bottom:170px;
+    left: 700px;
+}
+
+#login-button{
+    width:100px;
+    height: 50px;
+    color: white;
+    font-size: 22px;
+    font: "Helvetica Neue";
+    background: linear-gradient(#6491c4, #1c5380);
+    border:1px solid #06365f;
+    box-shadow: 0 3px 3px #1c5380;
+    position:relative;
+    bottom: 250px;
+    left: 1150px;
```

Comment:
Anytime you do position:relative and then you give it 1,000 pixels in one direction, there's a better way to move the element.  
It looks fine on the page but you should explore other options.